assets/js/helpers/urlUtils.js

```diff
@@ -12,6 +12,25 @@ export function isValidHttpUrl(urlString) {
 		return false;
 	}
 
+	// Prevents incorrect resource submission
+	let pathname = url.pathname;
+	// https://example.com -> pathname = /
+	// https://example.com/ -> pathname = /
+	// So, more than / is required if there is something behind the domain, like https://example.com/with-something-behind.ext
+	if(pathname !== '/') {
+		// But there may also be multiple nested directories, like... 
+		// https://example.com/1/2/3/file.ext (extension needs to be checked)
+		// https://example.com/1/2/3/ (always valid)
+		const endPathname = pathname.substring(pathname.lastIndexOf('/') + 1);
+		// There is an extension after the lastest slash (presence of a dot)? 
+		if(endPathname.indexOf('.') !== -1) {
+			if(endPathname.match(/\.(DTD|htm|html|xhtml|xht|mht|mhtml|maff|asp|aspx|adp|bml|cfm|cgi|ihtml|jsp|lass|lasso|lassoapp|pl|php|shtml|stm)$/i)) {
+				return url.protocol === "http:" || url.protocol === "https:";
+			}
+			return false;
+		}
```

Comment:
J'aurais tendance à simplifier le code de cette façon pour éviter le code dupliqué

```suggestion
		if(endPathname.indexOf('.') !== -1) && (!endPathname.match(/\.(DTD|htm|html|xhtml|xht|mht|mhtml|maff|asp|aspx|adp|bml|cfm|cgi|ihtml|jsp|lass|lasso|lassoapp|pl|php|shtml|stm)$/i)) {
			return false;
		}
```